app/src/main/java/ufrj/devmob/votadevmob/main/MainActivity.kt

```diff
@@ -23,7 +24,8 @@ class MainActivity : AppCompatActivity(), MainContract.View {
 
         presenter = MainPresenter(this)
 
-        createPollButton.setOnClickListener { showToastError(getString(R.string.main_create_poll_button)) }
+        val poll = Poll()
+        createPollButton.setOnClickListener { goToActivity(NewPollActivity::class.java, poll) }
```

Comment:
Você pode só usar o `startActivity(Intent(this, NewPollActivity::class.java)` direto, o `goToActivity` foi feito pro presenter poder chamar, se vc já está na activity não precisa. Com isso vc tbm n precisa criar um poll vazio só pra chamar o método.